src/components/additem/ProductPrice.jsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./ProductPrice.module.css";
+import commonStyles from "./AddItemCommon.module.css";
+
+const ProductPrice = ({ productPrice, setProductPrice }) => {
```

Comment:
P2:
컴포넌트명과 대소문자여부만 차이가 나니 헷갈리네요.
ProductPrice 컴포넌트에서 사용되는 props니 아래와 같은 이름을 추천드릴께요.

```suggestion
const ProductPrice = ({ value, onChange }) => {
```